src/t8_geometry/t8_geometry_implementations/t8_geometry_occ.cxx

```diff
@@ -336,8 +340,8 @@ t8_geometry_occ::t8_geom_evaluate_occ_triangle (t8_cmesh_t cmesh, t8_gloidx_t gt
         /* Calculate displacement between points on curve and point on linear curve.
          * Then scale it and add the scaled displacement to the result. */
         for (int dim = 0; dim < 3; ++dim) {
-          double displacement = pnt.Coord (dim + 1) - glob_intersection[dim];
-          double scaled_displacement = displacement * pow (scaling_factor, 2);
+          const double displacement = pnt.Coord (dim + 1) - glob_intersection[dim];
+          const double scaled_displacement = displacement * scaling_factor * scaling_factor;
```

Comment:
How could this happen in the first place?! m(

Reply:
We already had quite a lot of review rounds done, and performance engineering was not the top priority, since I knew that I will do these optimizations while implementing the batch processing